test_items.py

```diff
@@ -0,0 +1,94 @@
+import pytest
+import trello_items as trello
+import Item as it
```

Comment:
The file should be item.py so that the module is item with a lowercase i. The class will be Item with an uppercase i.

Then you want to do `from item import Item`

And replace your `it.Item` calls below with just `Item`. Similarly in trello_items.py

Reply:
done